templates/pages/admin/form/form_question.html.twig

```diff
@@ -73,6 +75,15 @@
             </div>
             {# Display question's title #}
             <div class="d-flex mt-n1 align-items-center">
+                <i
+                    class="ti ti-alert-triangle text-warning me-2 {{ is_anonymous and not question_type.isAllowedForAnonymousForm() ? '' : 'd-none' }}"
+                    data-glpi-form-editor-blacklisted-question-type-warning
+                    data-bs-toggle="popover"
+                    data-bs-trigger="hover"
+                    data-bs-placement="top"
+                    data-bs-html="true"
+                    data-bs-content="{{ __("This question type cannot be used in a public form. User must be authenticated to view this question type. Check your access policy.") }}"
```

Comment:
This text is ambiguous. I first thought that it would not be possible to add this question.

```suggestion
                    data-bs-content="{{ __("The current access policy allows unauthenticated access to this form, but this question type will be hidden to unauthenticated users.") }}"
```